dashboard/app/helpers/survey_results_helper.rb

```diff
@@ -1,24 +1,16 @@
 module SurveyResultsHelper
   def show_survey?
-    # rubocop:disable Lint/UnreachableCode
-
-    # Disabled for all users on 2016-03-30 after being shown for thirty days.
-    return false
-
     # Reasons we would not show the survey.
     unless current_user
       return false
     end
-    unless current_user.teacher?
-      return false
-    end
     if current_user.under_13?
       return false
     end
     if language != "en"
       return false
     end
-    if SurveyResult.where(user_id: current_user.id).exists?
+    if SurveyResult.where({user_id: current_user.id, kind: "NetPromoterScore2017"}).exists?
```

Comment:
Any chance `NetPromoterScore2017` can be made a constant in the `SurveyResult` model and referenced here?